app/src/androidTest/java/org/mozilla/fenix/ui/robots/NavigationToolbarRobot.kt

```diff
@@ -45,6 +46,24 @@ class NavigationToolbarRobot {
 
         val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
 
+        fun clearAddressBarActions(interact: BrowserRobot.() -> Unit): BrowserRobot.Transition {
```

Comment:
I would say  to rename this fun aas  clearAdddressBar... there are no more actions and it was  confusing for me first time looking at it. Just my opinion...

Reply:
Yes, I will try to split it